src/ecs/resources/mod.rs

```diff
@@ -11,6 +11,6 @@ mod broadcaster;
 
 pub use self::broadcaster::Broadcaster;
 pub use self::camera::{Camera, Projection};
-pub use self::input::{Axis, Button, InputHandler, PressedButtons, PressedKeys, PressedMouseButtons};
+pub use self::input::{Axis, Button, Buttons, KeyCodes, MouseButtons, InputHandler};
```

Comment:
Sorry, but this is not ordered alphabetically. Hope I don't annoy you with that.